platform-sdk/swirlds-merkle/src/testFixtures/java/com/swirlds/merkle/test/fixtures/map/lifecycle/SaveExpectedMapHandler.java

```diff
@@ -18,15 +18,22 @@
 
 import static com.swirlds.logging.legacy.LogMarker.EXCEPTION;
 
-import com.fasterxml.jackson.annotation.JsonIgnore;
```

Comment:
Please ignore these PTT related changes. Maintaining this code is a waste of time because we want to delete it as soon as possible.